processors/tenantidprocessor/tenantidprocessor.go

```diff
@@ -63,3 +90,60 @@ func (p *processor) addTenantIdToSpans(traces pdata.Traces, tenantIDHeaderValue
 		}
 	}
 }
+
+func (p *processor) addTenantIdToMetrics(metrics pdata.Metrics, tenantIDHeaderValue string) {
+	rms := metrics.ResourceMetrics()
```

Comment:
Rather than add the tenant_id to each instrumentation metric, we should just add it to the resource.

We should also make this change for spans too, where we simply add it to the resource.

Reply:
When we add tenantId to resourceAttributes for metrics, they are lost as prometheus-exporter only reads and accumulates instrumentationLibraryMetrics https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/prometheusexporter/accumulator.go#L66. So we need to add to each instrumentation metric.

When we add tenantId to resourceAttributes for traces, in spans it does not show up in tags, but it shows up in `Process` like :
![image](https://user-images.githubusercontent.com/74622220/112369433-3004bd00-8d02-11eb-9240-23b711e24f0e.png)